security/tor/src/opnsense/service/templates/OPNsense/Tor/torrc

```diff
@@ -54,16 +54,14 @@ SOCKSPolicy {{ policy.action }}{% if policy.type == 'v6' %}6{% endif %} {{ polic
 SOCKSPolicy reject *
 SOCKSPolicy reject6 *
 
-
```

Comment:
For better or worse, it's better to keep the extra lines than to push them in a feature addition